src/app/templates/DelegateForm.tsx

```diff
@@ -64,12 +64,13 @@ const sponsoredBaker = 'tz1aRoaRhSpRYvFdyvgWLL6TGyRoGF51wDjM';
 const DelegateForm: FC = () => {
   const { registerBackHandler } = useAppEnv();
   const formAnalytics = useFormAnalytics('DelegateForm');
+  const network = useNetwork();
 
   const acc = useAccount();
   const tezos = useTezos();
 
   const accountPkh = acc.publicKeyHash;
-  const assetSymbol = 'ꜩ';
+  const assetSymbol = network.type === 'dcp' ? 'ф' : 'ꜩ';
 
   const { data: balanceData, mutate: mutateBalance } = useBalance('tez', accountPkh);
```

Comment:
does this work correct with `useBalance('tez'` ?

Reply:
yes